<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordsLesson</class>
 <widget class="QWidget" name="WordsLesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>91</y>
     <width>1279</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>If you haven’t already gone and messed with the translator page to try forming words in morse code, we encourage you to go try now. You’ll see slashes “/” in between words, which indicates you waited long enough before you started inputting the next morse character, so it started the next word.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="font">
     <font>
      <family>OldNewspaperTypes</family>
     </font>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Morse code has an international set of timings, which determines how long dots, dashes, spaces between dots and dashes (intra-character spaces), spaces between different english characters (inter-character spaces), and spaces between words are.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>1141</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>OldNewspaperTypes</family>
       <pointsize>22</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Most often, you’ll encounter these timings, which are also the timings for this app:
Dot: 1 unit
Dash: 3 units
Intra-character space: 1 unit
Inter-character space: 3 units
Space between words: 7 units

This means that, for the word “paris”, the timings will add up to 50 units in total.
.--. .- .-. .. … /
10 dots = 10 units; 4 dashes = 12 units; 9 intra-character spaces = 9 units; 4 inter-character spaces = 12 units; and 1 word space at the end = 7 units; 10 + 12 + 9 + 12 + 7 = 50 units
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>1281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-top: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="startPracticeButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>OldNewspaperTypes</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Click To
Start
Practice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSlideButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="previousSlideButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-bottom: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old London</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lesson 9: Words</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
